tests/integration/test_db_admin.py

```diff
@@ -45,7 +45,7 @@ async def test_landscape_scalable_bundle_db(ops_test: OpsTest, charm: str) -> No
         application_name=DATABASE_APP_NAME,
         num_units=DATABASE_UNITS,
         base=CHARM_BASE,
-        config={"profile": "testing", "plugin_plpython3u_enable": "True"},
```

Comment:
No longer necessary?

Reply:
The exact plugins will be added and logged when we patch the bundle, so we only need the profile here.